### PR TITLE
Feature/move forgotten dx components

### PR DESCRIPTION
**Pull request by dramoturg**

### CODE REVIEW

**Comment by raveclassic** [comment]
Please, don't include old DatePicker/Calendar components

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (raveclassic) [inline .idea/codeStyleSettings.xml]: don't include this

File: packages/react-kit/src/utils/date.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { Moment } from 'moment';
```
Comment (raveclassic) [inline packages/react-kit/src/utils/date.ts]: no moment in project please

Comment (dramoturg) [reply packages/react-kit/src/utils/date.ts]: done

**Pull request merged**